### ISSUE TITLE
why the teacher net uses log_softmax?

### ISSUE THREAD

**Issue by DingLei14** [PolarisShi/distillation#1]

**Comment by JimmyDengJJ** [comment]
You can refer to the definition/document of PyTorch's KL Divergence loss (KLDivLos).
As with :class:`~torch.nn.NLLLoss`, the `input` given is expected to contain
    *log-probabilities* and is not restricted to a 2D Tensor.
    The targets are given as *probabilities* (i.e. without taking the logarithm).
